### PR TITLE
Types smoke test

### PR DESCRIPTION
**Pull request by mgrybyk**
Smoke tests for types to make sure all of them can be imported in mocha, jest and jasmine.

Should prevent issues like https://github.com/webdriverio/expect-webdriverio/pull/104 https://github.com/webdriverio/expect-webdriverio/pull/64 https://github.com/webdriverio/expect-webdriverio/pull/63

### CODE REVIEW

File: test-types/copy.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const fs = require('fs')
```
Comment (christian-bromann) [inline test-types/copy.js:1]: Should we move this file into a `/scripts` folder maybe (just to stay consistent with the main repo)?

File: test-types/copy.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        cp('-r', 'types', destDir)
+    }
+}
+
+/**
+ * delete eventual artifacts from test folders
+ */
+Promise.all(
+    artifactDirs.map((dir) =>
+        Promise.all(outDirs.map((testDir) => promisify(rimraf)(path.join(__dirname, testDir, dir))))
```
Comment (christian-bromann) [inline test-types/copy.js:62]: Let's define `promisify(rimraf)` somewhere at the top.

File: test-types/types.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+expect({}).toBeDisabled()
```
Comment (christian-bromann) [inline test-types/types.ts:1]: Shouldn't `expect` expect an element object here?

File: test-types/copy.js (hunk 1 above)
Comment (mgrybyk) [reply test-types/copy.js:1]: do you think it worth creating `tests` folder as well and then moving `test` and `test-types` inside?
```
tests
 - unit (or just test?)
 - types
```

File: test-types/copy.js (hunk 2 above)
Comment (mgrybyk) [reply test-types/copy.js:62]: I just copy pasted the script from main repo, but yeah, sounds good

File: test-types/types.ts (hunk 3 above)
Comment (mgrybyk) [reply test-types/types.ts:1]: nope, it expects anything

File: test-types/copy.js (hunk 1 above)
Comment (mgrybyk) [reply test-types/copy.js:1]: copy.js is in tests folder in the main repo - https://github.com/webdriverio/webdriverio/blob/master/tests/typings/copy.js

Comment (christian-bromann) [reply test-types/copy.js]: > do you think it worth creating tests folder as well and then moving test and test-types inside?

Yes!

> copy.js is in tests folder in the main repo

Ooops 🙈 nevermind then

**Pull request merged**